package.json

```diff
@@ -5,12 +5,11 @@
   "description": "AWS SDK for JavaScript from the future",
   "main": "index.js",
   "scripts": {
+    "generate-clients": "node ./scripts/generate-clients",
     "bootstrap": "yarn",
     "clean": "yarn clear-build-cache && yarn clear-build-info && lerna clean",
     "clear-build-cache": "rimraf ./packages/*/build/* ./clients/*/*/build/*",
     "clear-build-info": "rimraf ./packages/*/*.tsbuildinfo ./clients/*/*/*.tsbuildinfo",
-    "copy-models": "node ./scripts/copyModels.js",
-    "update-clients": "node ./packages/package-generator/build/cli.js import-all --matching './models/*/*/service-2.json'",
     "build:crypto-dependencies": "lerna run --scope '@aws-sdk/types' --scope '@aws-sdk/util-utf8-browser' --scope '@aws-sdk/util-locate-window' --scope '@aws-sdk/hash-node' --include-filtered-dependencies pretest",
```

Comment:
lerna complains about --include-filtered-dependencies being deprecated.  This could change this to --include-dependencies if we want to bump lerna.

Reply:
Will do